elements/pfe-sass/mixins/_components.scss

```
@@ -246,3 +246,84 @@
     }
   }
 }
+
+/// ===========================================================================
+/// List styles
+/// ===========================================================================
+
+// Mixins and extends can't use the LOCAL-VARIABLES map because we would be
+// at risk of overriding an elements map if they call our mixin. We need to
+// use just regular sass variables and pfe-local and specify the $fallbacks.
+@mixin ordered-list-primary($size: 56px) {
+  // First we create our Sass variables that we will use for fallbacks
+  // The gap between list items
+  $gap: pfe-var(content-spacer);
+  // The space between the circle and content
+  $spacer: pfe-var(content-spacer);
+  // rgb value of the color theme of the circle
+  // we need to wrap this in a calc function so it can be dynamic.
+  $color: (238, 0, 0);
+  $font-size: $pf-global--FontSize--2xl;
```

```suggestion
  $number-font-size: $pf-global--FontSize--2xl;
  $font-size:  $pf-global--FontSize--xl;
```
Suggested addingseparate variables for numbers vs list item text which needs to be 20px

---

Updated a more descriptive variable name for targeting the list number font size.